File: onboarding-servey/src/main/java/com/innercircle/onboardingservey/domain/Survey.java
package com.innercircle.onboardingservey.domain;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.RequestMapping;

@Entity
@Table(name = "serveys")
@RequiredArgsConstructor
@Getter
public class Survey {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long surveyId;
    private final String title;
    private final String description;

}

File: src/main/kotlin/org/inner/circle/survey/domain/Survey.kt
package org.inner.circle.survey.domain

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "survey")
class Survey(
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long = 0,
    @Column(nullable = false)
    var title: String,
    @Column(nullable = true)
    var description: String,
    @Column(nullable = false)
    var writer: String,
) : BaseEntity() {
    constructor() : this(title = "", description = "", writer = "")
}
